src/analyses/extractPthread.ml

```diff
@@ -132,11 +132,9 @@ module Tbls = struct
       Hashtbl.find table k |> Option.default_delayed new_value_thunk
 
 
-    let get_key v =
-      table |> Hashtbl.filter (( = ) v) |> Hashtbl.keys |> Enum.get
+    let get_key v = table |> Hashtbl.to_seq |> Seq.find_map (fun (k,v') -> if v' = v then Some k else None)
```

Comment:
Should leave a TODO here about the inefficiency of looking up by value from a `Hashtbl`.